test/io/test_big_schema.py

```diff
@@ -70,3 +70,23 @@ def test_should_not_fail_with_stack_overflow(defaultenv):
         assert response.status_code == 404
         data = response.json()
         assert data["code"] == "PGRST205"
+
+
+def test_second_request_for_non_existent_table_should_be_quick(defaultenv):
+    "requesting a non-existent relationship the second time should be quick"
+
+    env = {
+        **defaultenv,
+        "PGRST_DB_SCHEMAS": "apflora",
+        "PGRST_DB_POOL": "2",
+        "PGRST_DB_ANON_ROLE": "postgrest_test_anonymous",
+    }
+
+    with run(env=env, wait_max_seconds=30) as postgrest:
+        response = postgrest.session.get("/unknown-table?select=unknown-rel(*)")
+        assert response.status_code == 404
+        data = response.json()
+        assert data["code"] == "PGRST205"
+        first_duration = response.elapsed.total_seconds()
+        response = postgrest.session.get("/unknown-table?select=unknown-rel(*)")
+        assert response.elapsed.total_seconds() < first_duration / 20
```

Comment:
Is it necessary to use resource embedding?

```suggestion
        response = postgrest.session.get("/unknown-table")
        assert response.status_code == 404
        data = response.json()
        assert data["code"] == "PGRST205"
        first_duration = response.elapsed.total_seconds()
        response = postgrest.session.get("/unknown-table")
        assert response.elapsed.total_seconds() < first_duration / 20
```

Reply:
> Is it necessary to use resource embedding?

No, copy/paste leftover. Fixed.